magicctapipe/scripts/lst1_magic/semi_automatic_scripts/job_accounting.py

```diff
@@ -0,0 +1,178 @@
+"""
+This script does checks of status of jobs based on the log files generated during the execution.
+It also does accounting of memory and CPU usage
+"""
+import argparse
+import glob
+from datetime import timedelta
+from subprocess import PIPE, run
+
+import numpy as np
+import yaml
+
+from magicctapipe import __version__
+
+GREEN = "\033[32m"
+YELLOW = "\033[33m"
+RED = "\033[31m"
+ENDC = "\033[0m"
+
+
+def run_shell(command):
+    """
+    Simple function to extract the output of a command run in a shell
+
+    Parameters
+    ----------
+    command : str
+        Command to be executed
+
+    Returns
+    ----------
+    list
+        List of lines returned by the program
+    """
+    result = run(command, stdout=PIPE, stderr=PIPE, shell=True, universal_newlines=True)
+    return result.stdout
+
+
+def main():
+    """
+    Function counts the number of jobs that should have been submitted, and checks the output of the logs to see how many finished successfully, and how many failed.
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--config-file",
+        "-c",
+        dest="config_file",
+        type=str,
+        default="./config_general.yaml",
+        help="Path to a configuration file config_general.yaml",
+    )
+
+    parser.add_argument(
+        "--data-level",
+        "-d",
+        dest="data_level",
+        type=str,
+        default="DL1/M1",
+        help="Data level to be checked",
+    )
+
+    parser.add_argument(
+        "--version",
+        "-v",
+        dest="version",
+        type=str,
+        default=__version__,
+        help="MCP version (used for subdirectory name)",
+    )
+
+    args = parser.parse_args()
+    with open(args.config_file, "r") as f:
+        config = yaml.safe_load(f)
+
+    # TODO: those variables will be needed when more features are implemented
+    # source_in = config["data_selection"]["source_name_database"]
+    # source_out = config["data_selection"]["source_name_output"]
+    # timerange = config["data_selection"]["time_range"]
+    # skip_LST = config["data_selection"]["skip_LST_runs"]
+    # skip_MAGIC = config["data_selection"]["skip_MAGIC_runs"]
+    work_dir = config["directories"]["workspace_dir"]
+
+    print(f"Checking progress of jobs stored in {work_dir}")
+    dirs = sorted(glob.glob(f"{work_dir}/v{args.version}/*/{args.data_level}/*/*"))
```

Comment:
if there are no directories found there are no successful jobs either, so summary is simply empty. I made a change now also to fix the two issues above - if the directory is empty it will list the existing version directories and compatible data types, e.g.: 
```
job_accounting -c config_general.yaml
Checking progress of jobs stored in /home/julian.sitarek/ws/analiza/autoMCP/20240513/out_no_match1/
Error, no directories found
for path /home/julian.sitarek/ws/analiza/autoMCP/20240513/out_no_match1/ found in config_general.yaml this is available
Versions ['0.4.2.dev138+g6076d23.d20240510']
Supported data types: DL1/Observations/M1, DL1/Observations/M2
```
and script exits with error code 1